datasets/slides/glass.rmd

```diff
@@ -0,0 +1,147 @@
+---
+title: "Glass Dataset"
+output:
+  bookdown::pdf_document2:
+    citation_package: default
+    latex_engine: xelatex
+    toc: false
+---
+
+# Introduction
+```{r, warning = FALSE, message = FALSE, echo = FALSE}
+# import necessary libraries
+library(skimr)
+library(ggplot2)
+library(gridExtra)
+library(reshape2)
+library(lares)
+library(tidyverse)
+library(mlbench)
+library(corrplot)
+theme_set(theme_bw())
+```
+
+The glass identification dataset was created by B. German, Central Research Establishment, Home Office Forensic Science Service, Aldermaston, Reading, Berkshire RG7 4PNA. The data frame with 214 observations has examples of the chemical analysis of 7 varieties of glass. The target is to predict the kind of glass type based on the chemical analysis.
+
+(ref:glass-cap) Source: macrovector ([link](https://www.freepik.com/free-vector/empty-realistic-glasses-set-different-alcohol-drinks-cocktails-isolated_10347034.htm#query=glass&position=4&from_view=search))
```

Comment:
this image doesn't really represent the target label, does it?

Reply:
No, this image is just for a nice visualization, like the Boston images in the Boston dataset.